src/Integration/Connection/UI/OrganizationSelectionWindow.xaml

```diff
@@ -41,46 +42,98 @@
+                    <RowDefinition SharedSizeGroup="tabItemSize" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition SharedSizeGroup="tabItemHeight" />
+                </Grid.ColumnDefinitions>
+
+                <StackPanel 
+                    Style="{StaticResource DialogContent}">
+                    <TextBlock Margin="0,0,0,8"
+                FontStyle="Italic" Text="Please select the organization you want to bind to." TextWrapping="Wrap" />
+
+                    <StackPanel Orientation="Horizontal">
+                        <TextBlock VerticalAlignment="Center" Text="Organizations" />
+                        <ComboBox x:Name="OrganizationComboBox"
+                    Width="250" Margin="12,0,0,0"
+                    HorizontalContentAlignment="Stretch"
+                    IsEditable="True" IsTextSearchCaseSensitive="False" IsTextSearchEnabled="True"
+                    TextSearch.TextPath="Name">
+                            <ComboBox.ItemsPanel>
+                                <ItemsPanelTemplate>
+                                    <VirtualizingStackPanel />
+                                </ItemsPanelTemplate>
+                            </ComboBox.ItemsPanel>
+                            <ComboBox.ItemContainerStyle>
+                                <Style TargetType="{x:Type ComboBoxItem}">
+                                    <Setter Property="Width" Value="230" />
+                                </Style>
+                            </ComboBox.ItemContainerStyle>
+                            <ComboBox.ItemTemplate>
+                                <DataTemplate DataType="{x:Type models:SonarQubeOrganization}">
+                                    <TextBlock TextTrimming="CharacterEllipsis">
+                        <Run Text="{Binding Name, Mode=OneWay}" />
+                        <Run Text="(" /><Run Text="{Binding Key, Mode=OneWay}" /><Run Text=")" />
+                                    </TextBlock>
+                                </DataTemplate>
+                            </ComboBox.ItemTemplate>
+                        </ComboBox>
+                    </StackPanel>
+
+                    <Separator Style="{StaticResource DialogSeparator}" />
+
+                    <StackPanel VerticalAlignment="Center" Style="{StaticResource DialogButtonsPanel}">
+                        <vsui:DialogButton Click="OnOwnOkButtonClick" Content="OK" IsDefault="True"
+                                IsEnabled="{Binding ElementName=OrganizationComboBox,
+                                                    Path=SelectedItem,
+                                                    Converter={StaticResource NotNullConverter}}" />
+                        <vsui:DialogButton Content="Cancel" IsCancel="True" />
+                    </StackPanel>
+                </StackPanel>
+
+            </Grid>
+        </TabItem>
+        <TabItem Header="Other organizations">
```

Comment:
This is the new tab containing a textbox for the use to type in a organization key.